src/bash/autostartdaemon.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+###
+# Helper script to setup an autostart daemon for openDash
+###
+WorkingDirectory="/home/pi/dash/"
+if [[ $1 != "" ]]
+then
+    WorkingDirectory="/home/pi/$1/dash/"
+fi
+echo ${WorkingDirectory}
+
+#stop and disable dash service
+sudo systemctl stop dash.service
```

Comment:
does this work if there was no dash service previously?

Reply:
@rsjudka Yes, doesn't throw any errors

Reply:
this is what i see on a system with no dash service:
```
$ sudo systemctl stop dash.service
Failed to stop dash.service: Unit dash.service not loaded.
```